crates/cranelift/src/wasmfx/optimized.rs

```diff
@@ -460,11 +463,6 @@ pub(crate) mod typed_continuation_helpers {
                     let new_revision = self.get_revision(env, builder);
                     emit_debug_assert_eq!(env, builder, revision_plus1, new_revision);
                 }
-                let overflow =
-                    builder
-                        .ins()
-                        .icmp_imm(IntCC::UnsignedLessThan, revision_plus1, 1 << 16);
-                builder.ins().trapz(overflow, ir::TrapCode::IntegerOverflow); // TODO(dhil): Consider introducing a designated trap code.
```

Comment:
I'd like to preserve these traps in debug mode. I think that can prove useful.

Reply:
And what should they check?

Reply:
They should check whether the revision counter has wrapped around.